applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.cpp

```diff
@@ -45,6 +46,9 @@ void KratosCompressiblePotentialFlowApplication::Register()
     KRATOS_REGISTER_VARIABLE(VELOCITY_POTENTIAL);
     KRATOS_REGISTER_VARIABLE(AUXILIARY_VELOCITY_POTENTIAL);
 
+    //Embedded variables
+    KRATOS_REGISTER_VARIABLE(LEVEL_SET);
```

Comment:
You need this variable because you are working with two level-set functions at the same time (body and wake). Am I wrong?

Reply:
Yes, you are right! We need another level-set function for the wake.

Reply:
`LEVEL_SET`can be misleading. What do you think if we call it  `WAKE_DISTANCE` or `WAKE_LEVEL_SET`? My concern is that to have `DISTANCE` and `LEVEL_SET` inside the same element can be misunderstood by someone who is not used to your formulation.

Reply:
Actually, this `LEVEL_SET` variable is the one used to embed the geometry, so maybe I can rename it to `GEOMETRY_DISTANCE`/`GEOMETRY_LEVEL_SET`?

Would you create variables for both cases so that it's clear what each variable is doing?

Current naming vs proposed:
- Geometry: `LEVEL_SET` - > `GEOMETRY_DISTANCE`
- Wake: `DISTANCE` -> `WAKE_DISTANCE`

Reply:
@rubenzorrilla, what would you do with the variable naming?

We can discuss this further in the next PR if you prefer, where i will introduce the second DISTANCE function for the wake.

Reply:
I think that this is much more self-explained. If I'm not wrong, the current implementation of the distance calculation processes assumes that the variable in which the level set is saved is `DISTANCE`, so you need to compute it an the to copy it. Maybe we should change this to accept any variable...